product_docs/docs/pwr/1/rel_notes/110_rel_notes.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Postgres Workload Report 1.1.0 release notes"
+navTitle: Version 1.1.0
+---
+
+Released: 14 May 2024
+
+New features, enhancements, bug fixes, and other changes in Postgres Workload Report 1.1.0 include:
+
+|       Type        |                                            Description                                                                                                  |
+| ----------------- |-------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Feature    | Add locking to PWR commands to avoid concurrent runs |
+| Change     | Changed the default value for the `assets_dir` config parameter to use the folder where packages place the assets  (`/usr/share/pwr/assets`) |
+| Change     | Added the `--use-plain-report-name` option to `execute` and `report` commands. This will bypass the default behavior of prepending the database name and report timestamp to the report name |
```

Comment:
```suggestion
| Change     | Added the `--use-plain-report-name` option to `execute` and `report` commands. This will bypass the default behavior of appending the database name and report timestamp to the report name |
```